delfin/drivers/hpe/hpe_3par/component_handler.py

```diff
@@ -0,0 +1,248 @@
+# Copyright 2020 The SODA Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import time
+
+from oslo_log import log
+
+from delfin import exception
+from delfin.common import constants
+from delfin.drivers.hpe.hpe_3par import consts
+
+LOG = log.getLogger(__name__)
+
+
+class ComponentHandler():
+    """Hpe3par's Component handler，Superclass,
+    """
+    # ssh command
+    HPE3PAR_COMMAND_CHECKHEALTH = 'checkhealth'  # return: System is healthy
+    HPE3PAR_VERSION = 'Superclass'
+
+    HPE3PAR_VENDOR = 'HPE'
+
+    def __init__(self, restclient=None, sshclient=None):
+        self.restclient = restclient
```

Comment:
Do we need both rest client and ssh client ? are these both served by same instance ?